src/util/package-branch.ts

```diff
@@ -0,0 +1,20 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// map from a packageName to the prefix used in release branch creation.
+export function packageBranch(packageName: string) {
```

Comment:
not super thrilled with this approach but it felt better than copy/pasting [this](https://github.com/googleapis/release-please/pull/669/files#diff-dc4bacd1947218131d029b0447762952f50233b4a6c61d56ae416554556c3139L277) around

* better name?
* better scoping (per releaser somehow? right now this just handles specific node/npm remapping)

Reply:
@joeldodge79 I might call it `parsePackageName` and have it return a `scope`, and `name`, which are the names given to the two parts of the package name?

https://docs.npmjs.com/cli/v6/using-npm/scope

Reply:
I'm going to suggest another route. updated PR coming soon